src/ShearStressTransportEquationSystem.C

```diff
@@ -179,6 +180,11 @@ ShearStressTransportEquationSystem::solve_and_update()
     tkeEqSys_->compute_projected_nodal_gradient();
     sdrEqSys_->assemble_nodal_gradient();
     clip_min_distance_to_wall();
+
+    // FIXME: Figure out a way to do this properly in TAMS Eqn System, so that 
+    //        they are decoupled
+    if (realm_.solutionOptions_->turbulenceModel_ == SST_TAMS) 
+        initialize_mdot();
```

Comment:
Can we rename this to `initialize_average_mdot()`?

Reply:
done.